_includes/page.hbs

```diff
@@ -67,34 +51,21 @@ scripts: []
 </head>
 
 <body>
-  <article id="patreon-banner" class="patreon-banner">
-    <div>
-      <p>
-        lipu tenpo li lon ma Patreon!
-        <a href="https://www.patreon.com/liputenpo">o lukin!</a>
-      </p>
-      <button class="close" onclick="closeBanner()">&#10006;</button>
-    </div>
-  </article>
   <header>
-    <div id="banner"></div>
-    <a href="/" class="title">
-      <img src="/logo.png" alt="lipu tenpo logo" width="475" height="497" />
-      <h1>lipu tenpo</h1>
-    </a>
+    <div class="sewi-lipu">
+      <a href="/">
+        <img src="/logo.png" alt="lipu tenpo logo" class="logo" />
+      </a>
+      <div>
+        <h1><a href="/">lipu tenpo</a></h1>
+        <div class="subtitle"><a href="/sona#faq">li alasa e jan pali sin</a></div>
+      </div>
+    </div>
     <nav>
       <ul>
-        <li>
-          <a href="/">open</a>
-        <li>
-          <a href="/toki">toki</a>
-        </li>
-        <li>
-          <a href="/sona">sona</a>
-        </li>
-        <li>
-          <a href="/lipu-ante">lipu ante</a>
-        </li>
+        {{{ nav "/" "lipu" }}}
+        {{{ nav "/toki/" "toki" }}}
+        {{{ nav "/sona/" "sona" }}}
```

Comment:
I still prefer 2-3 word links because I think they can provide helpful context, especially for people who are new to the site. I also designed this specifically for multiple words (each with their own line) so I'm curious how this looks.

Reply:
Here's my main argument for making the link text longer: https://discord.com/channels/795288348024373289/796468535672963122/1278030295211442291

The current automatic deployment link works for me! That's really helpful @alifeee thanks for working on that!